yarn.lock

```diff
@@ -36,6 +36,42 @@
     lodash "^4.17.21"
     resize-observer-polyfill "^1.5.0"
 
+"@apollo/client@^3.4.3":
```

Comment:
should this file be pushed?

Reply:
yeah, always. It's better to have lock files